src/ts/transformers/har.ts

```diff
@@ -134,15 +134,15 @@ export function transformPage(harData: Har, pageIndex: number = 0): WaterfallDat
     });
 
   const marks = Object.keys(pageTimings)
-    .filter((k) => (typeof pageTimings[k] === "number" && pageTimings[k] >= 0))
+    .filter((k: keyof PageTimings) => (typeof pageTimings[k] === "number" && pageTimings[k] >= 0))
     .sort((a: string, b: string) => pageTimings[a] > pageTimings[b] ? 1 : -1)
-    .map((k) => {
-      const startRelative = pageTimings[k];
+    .map((k: "onContentload" | "onLoad") => {
+      const startRelative: number = pageTimings[k];
 
       doneTime = Math.max(doneTime, startRelative);
 
       return {
-        "name": `${k.replace(/^[_]/, "")} (${startRelative}ms)`,
+        "name": `${k.replace(/^[_]/, "")} (${(Math.round(startRelative * 10) / 10)} ms)`,
```

Comment:
This is be a good place to use roundNumber (from misc.ts). Maybe no decimals are needed. Millisecond precision is small enough (it's also what WPT uses).